grpc/proto/v1/properties.proto

```diff
@@ -0,0 +1,30 @@
+
+syntax = "proto3";
+
+package weaviate.v1;
+
+option go_package = "github.com/weaviate/weaviate/grpc/generated;protocol";
+option java_package = "io.weaviate.client.grpc.protocol.v1";
+option java_outer_classname = "WeaviateProtoProperties";
+
+
```

Comment:
Can we remove this 1 empty line, so that only 1 (not 2) empty lines will be here